src/components/BorrowerProfile/LendCta.vue

```diff
@@ -309,6 +323,14 @@ export default {
 	},
 	data() {
 		return {
+			loan: {
```

Comment:
@egoroza The new LendAmountButton no longer has a dependency on the full loan being passed as prop so it can be removed from that file. Could you please work on that clean up and then go through and remove this extra loan setup that was added to this component?